index.html

```diff
@@ -5045,103 +5045,149 @@ <h2>DID Documents</h2>
 }
       </pre>
 
-      <pre class="example" title="DID Document with many different verification methods">
+      <pre class="example" title="DID Document with many different key types">
 {
   "@context": [
     "https://www.w3.org/ns/did/v1",
-    "https://w3id.org/security/suites/jws-2020/v1",
-    "https://w3id.org/security/suites/ed25519-2020/v1"
+    "https://w3id.org/security/suites/jws-2020/v1"
```

Comment:
I updated this second example to mirror the first example. 

The ordering still implies that multibase is better to use, but since the section is informative, I am not concerned about that.